lumicks/pylake/detail/widefield.py

```diff
@@ -364,13 +375,23 @@ def __init__(self, tiff_file, align_requested):
                 self.json[f"Channel {j} alignment"] = self.json.pop(f"Alignment {color} channel")
                 self.json[f"Channel {j} detection wavelength (nm)"] = "N/A"
 
-        if "Channel 0 alignment" in self.json:
-            self._alignment_matrices = {
-                channel: TransformMatrix.from_alignment(
-                    self._raw_alignment_matrix(channel), *self.offsets
-                ).invert()
-                for channel in range(3)
-            }
+        excitation_colors = [
+            key.split(" ")[0] for key in self.json.keys() if "Excitation Laser wavelength" in key
+        ]
+
+        # For two-color tiffs, the channel order may be different. Green is always channel 1
+        # and used as reference. The other channel will be stored in channel zero.
```

Comment:
I guess this is dictated by BL and there is no other way to get the channel order information directly from BL!?

Reply:
Yeah, actually, I want to verify this on a system this week.

The reference image is always the one in `channel[1]` (second channel). That makes the transformation matrices always in the order of the raw tiff channels.

In terms of relevant information, we have the following information available:
```
{'Alignment region of interest (x, y, width, height)': [0, 0, 2048, 682],
 'Blue Excitation Laser power level (%)': 2.0,
 'Blue Excitation Laser power on': 'true',
 'Blue Excitation Laser wavelength (nm)': 488,
 'Camera': 'WT',
 'Channel 0 alignment': [1.001670002937317,
  0.01721999980509281,
  -4.593649864196777,
  -0.017550000920891762,
  1.0012500286102295,
  33.25389099121094],
 'Channel 0 detection wavelength (nm)': '600/50',
 'Channel 1 alignment': [1.0, 0.0, 0.0, 0.0, 1.0, 0.0],
 'Channel 1 detection wavelength (nm)': '525/40',
 'Green Excitation Laser power level (%)': 2.0,
 'Green Excitation Laser power on': 'true',
 'Green Excitation Laser wavelength (nm)': 561,
```

Initially, I was under the impression that green would always have the unit transformation (but given the metadata I'm not so sure that is correct anymore).

If I look at the metadata that comes with the channels, I see that `Blue` is the one with the unit transform. I will add an extra commit that pulls the detection wavelengths from the BL metadata and uses those. I'd rather not hardcode the values, so I will order it by wavelength.